<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-BD50LHC" ignored="0" name="Test3" tests="1" failures="1" timestamp="2022-03-04T12:20:50 IST" time="33.362" errors="0">
  <testcase name="validateTitle" time="1.022" classname="testcases.ValidateTitleCase">
    <failure type="java.lang.AssertionError" message="expected [ ] but found [Login | Salesforce]">
      <![CDATA[java.lang.AssertionError: expected [ ] but found [Login | Salesforce]
at testcases.ValidateTitleCase.validateTitle(ValidateTitleCase.java:37)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- validateTitle -->
</testsuite> <!-- Test3 -->
